app/assets/stylesheets/helpers/_taxonomy-navigation.scss

```diff
@@ -14,6 +17,32 @@
   border-top: 1px solid $black;
 }
 
-.taxonomy-navigation__section:first-of-type {
-  margin-top: $gutter * 1.5;
+.taxonomy-navigation__row {
+  display: flex;
```

Comment:
Is there a reason why this needs to use flex rather than just the grid classes (`grid-row` and `column-one-third`)?

Reply:
So that they line up with the highlight boxes below.

Reply:
I think it can still line up without flex? Quick testing in the herokuapp looks ok:

<img width="1000" alt="screen shot 2018-07-31 at 15 06 37" src="https://user-images.githubusercontent.com/29889908/43464594-63989c10-94d3-11e8-872c-97f8833e3415.png">

Reply:
The breakpoints are different:

Grid:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/2715/43465149-a7f37e56-94d4-11e8-960e-0843977cb09c.gif)
Flex:

![ezgif com-video-to-gif-2](https://user-images.githubusercontent.com/2715/43465164-af37a782-94d4-11e8-87a8-145f2056fe6f.gif)

Reply:
Ah I see, I didn't spot that - that's annoying! Thanks for explaining. 

I wonder if we need it to match up or if we could just have each column half the width of that row. But using flex isn't a big deal, it just adds a lot of CSS.

Reply:
It was something @miaallers mentioned when we went through it together. I should imagine we can find a better way to implement if the AB test is positive.